### PR TITLE
fix testcafe show stoppers (close #148, close #149, close #150)

### LINKED ISSUE #148: Static ShadowUI functions don't available via hammerhead api
TestCafe uses `addClass` and `removeClass`, they are not available now via api.
This issue stops https://github.com/superroma/testcafe-phoenix/issues/44

### LINKED ISSUE #149: Selection sandbox doesn't work properly
`Selection` sandbox depends on `FocusBlurSandbox` but the instance of `FocusBlurSandbox` is `null` when `EventSandbox` creates the instance of `Selection` sandbox

The issue stops https://github.com/superroma/testcafe-phoenix/issues/44

### LINKED ISSUE #150: EventSandbox.focus function doesn't call the callback
when you try to focus some element and the current active element is an input element in an iframe.

Cause: incorrect `this` usage [here](https://github.com/superroma/testcafe-hammerhead/blob/master/src/client/sandbox/event/focus-blur.js#L308)

### PR DESCRIPTION
**Pull request by churkin**
\cc @inikulin @AlexanderMoskovkin

### CODE REVIEW

**Comment by churkin** [comment]
\cc @VasilyStrelyaev check comments pls

**Comment by AlexanderMoskovkin** [comment]
lgtm

File: src/client/sandbox/shadow-ui.js [hunk 1]
```diff
@@ -366,12 +366,20 @@ export default class ShadowUI extends SandboxBase {
     }
 
     // API
-    static addClass (el, value) {
+    // NOTE: this method cannot be static because it is part of the public API
```
Comment (VasilyStrelyaev) [inline src/client/sandbox/shadow-ui.js]: ...it is a part...

File: src/client/sandbox/shadow-ui.js [hunk 2]
```diff
@@ -366,12 +366,20 @@ export default class ShadowUI extends SandboxBase {
...
+    addClass (el, value) {
         var patchedClass = ShadowUI.patchClassNames(value);
 
         domUtils.addClass(el, patchedClass);
     }
 
+    // NOTE: this method cannot be static because it is part of the public API
```
Comment (VasilyStrelyaev) [inline src/client/sandbox/shadow-ui.js]: ...it is a part...

**Comment by churkin** [comment]
FPR

**Comment by inikulin** [comment]
lgtm

**Comment by VasilyStrelyaev** [comment]
lgtm

**Comment by AlexanderMoskovkin** [comment]
Lgtm

**Pull request merged**